docs/references/phpdoc/tags/param.rst

```diff
@@ -1,57 +1,53 @@
 @param
 ======
 
-The @param tag is used to document a single argument of a function or method.
+The ``@param`` tag is used to document a single `argument of a function or method`_.
 
 Syntax
 ------
 
+.. code-block::
+
     @param [Type] [name] [<description>]
 
 Description
 -----------
 
-With the @param tag it is possible to document the type and function of a
-single argument of a function or method. When provided it MUST contain a
-Type to indicate what is expected; the description on the other hand is
-OPTIONAL yet RECOMMENDED in case of complicated structures, such as associative
-arrays.
+With the ``@param`` tag it is possible to document the :doc:`Type <../types>`
+and function of a single `argument of a function or method`_. When provided it
+MUST contain a :doc:`Type <../types>` to indicate what is expected.
```

Comment:
I think we can indeed relax the `Type` to `MAY`, though I think `name` has to be required so that the parser can know how to tie a given param to what's in the code signature, in the case where some params are omitted in the docblock:

```
/**
 * I'm a FOO BAR function
 *
 * @param this array should contains strings... 
 *        but with no Type or Name given,
 *        how does phpDocumentor know I'm tied to $baz?
 * @return void
 */
public function foo (string $bar, array $baz, object $quz)
```